pkg/objcache/objcache.go

```diff
@@ -27,18 +27,20 @@ import (
 	"time"
 )
 
-// NoExpiry represents caches to be permanent and can only be deleted.
-var NoExpiry = time.Duration(0)
+const (
+	// NoExpiry represents caches to be permanent and can only be deleted.
+	NoExpiry = time.Duration(0)
 
-// DefaultExpiry represents default time duration value when individual entries will be expired.
-var DefaultExpiry = time.Duration(72 * time.Hour) // 72hrs.
+	// DefaultExpiry represents three days time duration when individual entries will be expired.
+	DefaultExpiry = time.Duration(3 * 24 * time.Hour)
 
-// DefaultBufferRatio represents default ratio used to calculate the
-// individual cache entry buffer size.
-var DefaultBufferRatio = uint64(10)
```

Comment:
It is kept public to be configurable as a generic package. Please keep it public.